active/InformationBar/InformationBar.md

```diff
@@ -0,0 +1,512 @@
+
+# Examples
+
+An info bar can have several configurations, here are some notable ones.
+
+When an InfoBar is conveying information of a common severity, like displaying an error or warning, it can be set to have one of many severity levels to use consistent [Fluent](https://docs.microsoft.com/en-us/windows/apps/fluent-design-system) styling for it's identifiers.
+![A mockup of an InfoBar in a Warning state with a close button and a message](images/Warning_NoButton.png)
+
+If a call to action is needed, an info bar can have customizable action or hyperlink buttons.
+![A mockup of an InfoBar](images/Default_MessageHyperlink.png)
+
+The info bar can also be customized with its optional properties and with XAML Content to include extra buttons and other UI elements.
+![A mockup of an InfoBar with custom status color and content](images/Custom_IconColor.png)
+
+## Create an InfoBar
+
+The XAML below describes an inline InfoBar with the default styling for an error  notification. An info bar can be created anywhere in the element tree or code behind. In this example the InfoBar expands to fill the width of the StackPanel layout control it resides in.
+
+By default, the InfoBar will not be visible. Set the IsOpen property to true in the XAML or code behind to display the information.
+
+XAML
+```xml
+<Grid>
+    <!--Other UI content -->
+    <StackPanel x:Name="ContentArea">
+        <controls:InfoBar x:Name="UnsuccessfulSaveNotification"
+            Severity="Warning"
+            Title="Error while saving"
+            Message="Your document was unable to be saved.">
+        </controls:InfoBar>
+    </StackPanel>
+    <!--Other UI content -->
+<Grid>
+```
+C#
+```C#
+public MainPage()
+{
+    this.InitializeComponent();
+
+    if(!SaveSuccessful())
+    {
+        UnsuccessfulSaveNotification.IsOpen = true;
+    }
+}
+```
+
+Here is the visual representation of the info bar in the page.
+
+![A mockup of an InfoBar in a Warning state with a close button and a message](images/Warning_NoButton.png)
+
+## Using pre-defined severity levels
+The type of the info bar can be set via the Severity property to automatically set a consistent status color, icon, and assistive technology settings dependent on the criticality of the notification.
+
+Mockups in various Severities to be added.
+![Mockup of InfoBar with no content in different Severity colors and icons for light mode](images/Docked_ToolkitSeverity.png)
+
+![Mockup of InfoBar with no content in different Severity colors and icons for dark mode](images/Docked_ToolkitSeverityDark.png)
+
+## Programmatic dismiss in info bar
+An info bar can be dismissed by the user via the close button or programmatically. If the notification is required to be in view until the status is resolved and you would like to remove the ability for the user to dismiss the info bar, you can set the IsUserDismissable property to false.
```

Comment:
"Close" is a more standard term for this action than "dismiss". And this looks more like a case for the "Can" pattern. So instead of "IsUserDismissable" I think "CanClose" would be a better name. (For example similar to [TextBox.CanUndo](https://docs.microsoft.com/uwp/api/Windows.UI.Xaml.Controls.TextBox.CanUndo).)

Reply:
I'm open to that, could we leave this as something to be discussed at the API Review?